qiskit/pulse/transforms/channel_transforms.py

```diff
@@ -0,0 +1,86 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+import numpy as np
+from typing import Union
+
+from base_transforms import target_qobj_transform
+from qiskit.providers.ibmq import IBMQBackend
+from qiskit.pulse import Schedule, Play, MeasureChannel
+from qiskit.pulse.channels import PulseChannel
+from qiskit.visualization.pulse_v2.events import ChannelEvents

```

Comment:
We likely should not be using ChannelEvents in the transforms module (circular imports). This may need to be moved, or is there another work around @nkanazawa1989?

Reply:
That was my intention. If we separate `ChannelEvents` from the visualizer, we can use that functionality for other purpose as well. Just porting the class from visualizer is not right way though it works. This approach may conflict with #5977

Reply:
I'm fine with deprecating `ChannelEvents` and writing new one in the transforms. This will drastically simplify the visualizer.